backend/routes/users.js

```diff
@@ -91,4 +91,31 @@ usersRouter.post(
     ),
 );
 
+usersRouter.post(
+    '/:userId/portfolios/:portfolioId/securities',
+    requestValidation({
+        params: {
+            portfolioId: joi.string(),
+            ...baseUserRequestValidationSchema.params
+        },
+        body: {
+            units: joi.string()
+                .allow('')
+                .optional(),
+            price: joi.string()
+                .allow('')
+                .optional(),
+            selectedPortfolio: joi.string()
```

Comment:
Given that the portfolio ID is specified in the route, we will not be needing this.